### ISSUE TITLE
Background registry editor

### ISSUE THREAD

**Issue by secbirkin** [amidaware/tacticalrmm#1452]
It would be very good to have a background registry editor fopr Windows clients.

**Comment by bbrendon** [comment]
@secbirkin Windows comes with two. reg.exe and in PowerShell New-ItemProperty

**Comment by wh1te909** [comment]
i think he means a web UI built into trmm, which is planned. we already have the interface for the file browser for reporting, same one will be used for registry editor

**Comment by secbirkin** [comment]
@wh1te909 yes, that's what I meant. A web GUI for registry editor like file browser. Thanks.

**Comment by Acwildweb** [comment]
Awesome! Hurry!!!

**Comment by wh1te909** [comment]
released in v1.4.0

**Comment by Acwildweb** [comment]
Hi, I’ve been testing the new registry editor, but I've noticed some mismatches between the keys shown here and those in regedit.exe. Specifically, some of my software’s keys are missing, even though I'm looking at the exact same path. It seems like I'm seeing fewer entries overall. Could you explain why this is happening and how I can view the full set of keys? Thanks for your help and your time.

**Comment by wh1te909** [comment]
@Acwildweb can you provide steps for us to reproduce, which software, windows version etc

**Comment by Acwildweb** [comment]
<img width="1791" height="758" alt="Image" src="https://github.com/user-attachments/assets/ab174f7f-d545-4daf-8cbd-f34f3bbabf87" />

<img width="5120" height="2636" alt="Image" src="https://github.com/user-attachments/assets/deebfdef-0d91-4e65-a866-7e18a452522c" />
As you can see on the two screenshot there are several differences
about the remote machine issued here all i nfo requested:
Name:Microsoft Windows 11 Pro|C:\Windows|\Device\Harddisk0\Partition3
Debug:false
Locale:0410
OSType:18
Status:OK
Caption:Microsoft Windows 11 Pro
CodeSet:1252
Primary:true
Version:10.0.26100

**Comment by wh1te909** [comment]
that's just the default value, it's not an actual entry

**Comment by Warming-nyit** [comment]
1. I have a test PC where it is possible to see TeamViewer entries through Windows Regedit.
This is not possible through Tactical

<img width="805" height="1217" alt="Image" src="https://github.com/user-attachments/assets/ab934704-be4f-461a-b590-afb9be0b7466" />

<img width="1292" height="910" alt="Image" src="https://github.com/user-attachments/assets/4c2e99e4-d9da-4b1d-b32c-ecbae104ee14" />


2. Under HKEY_CURRENT_USER\Software\, it looks like RegisteredApplications is missing in Tactical.

<img width="848" height="1056" alt="Image" src="https://github.com/user-attachments/assets/267e83b6-1647-46c1-8562-5a64e584ceae" />


3. If you type the entry URL in lowercase, it causes an error.

<img width="1262" height="114" alt="Image" src="https://github.com/user-attachments/assets/cc331897-d4fb-4175-acb3-d5ff4b5b2613" />


4. If you type an abbreviation in the entry URL (for example HKLM), it causes an error.

<img width="1190" height="104" alt="Image" src="https://github.com/user-attachments/assets/86f315d5-d020-4793-9b62-5a761d8e73e2" />

**Comment by Acwildweb** [comment]
> that's just the default value, it's not an actual entry

Hey, sorry, I don't get it. What do you mean it's not an actual entry? Are you saying the program doesn't show the real registry entries but just random ones?

**Comment by silversword411** [comment]
I can confirm there are missing keys in TRMM from what's on the computer. 

<img width="3266" height="1646" alt="Image" src="https://github.com/user-attachments/assets/6cd38975-3878-4fe2-9448-b65180424b51" />

**Comment by wh1te909** [comment]
Regarding inconsistencies in HKEY_CURRENT_USER, this is expected behavior because the agent runs under the SYSTEM account, so when you browse HKEY_CURRENT_USER you’re seeing SYSTEM’s user hive, not the currently logged-in user’s registry.

**Comment by silversword411** [comment]
I should have known better, yeah that's HKCU. You'd have to use the `/HKEY_USERS/whateverforauser` to edit HKCU

**Comment by Acwildweb** [comment]
I'm sorry, but I still don't understand. Perhaps the English-to-Italian translation is not very efficient. Could you give me a practical example of how to view the registry keys of the computer I'm connected to?

**Comment by wh1te909** [comment]
@Warming-nyit we've identified the issue for your teamviewer problem, root cause was a software name can be very long and didn't fit in the buffer size, a fix will be in the next release
